frontend/packages/kubevirt-plugin/src/utils/utils.ts

```diff
@@ -51,3 +52,20 @@ export const getVMLikeModelDetailPath = (
   namespace: string,
   name: string,
 ) => `${getVMLikeModelListPath(isCreateTemplate, namespace)}/${name}`;
+
+export const getVMWizardCreateLink = (
+  namespace: string,
+  itemName: VMWizardName = VMWizardName.WIZARD,
+  mode: VMWizardMode = VMWizardMode.TEMPLATE,
+  template?: string,
+) => {
+  // Overide mode if name is import.
+  const wizardMode = itemName === VMWizardName.IMPORT ? VMWizardMode.IMPORT : mode;
+
+  const type = itemName === VMWizardName.YAML ? '~new' : '~new-wizard';
+  const args = `${wizardMode ? `mode=${wizardMode}` : ''}${
```

Comment:
can we use URLSearchParams here?  as it is less prone to mistakes

Reply:
nice :+1: